mlflow/R/mlflow/DESCRIPTION

```diff
@@ -61,7 +61,7 @@ Suggests:
     testthat (>= 2.0.0),
     xgboost
 Encoding: UTF-8
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.2
```

Comment:
Can we revert this change?

Reply:
I am not sure why it was added to my branch but I reverted it